libautoscoper/src/Tracker.cpp

```diff
@@ -585,7 +599,21 @@ void Tracker::optimize(int frame, int dFrame, int repeats, int opt_method, unsig
 
 // Calculate Correlation for Bone Matching
 std::vector <double> Tracker::trackFrame(unsigned int volumeID, double* xyzypr) const
-  {
+{
+
+    // Upon a call to the Tracker::trackFrame() function, the following steps are performed:
+    //
+    // 1) Given a new pose, for each camera view:
+    //  a) Compute the modelView and inverseModelView matrices
```

Comment:
```suggestion
    //  a) Compute the modelView matrix
```

Looks like there are no `inverseModelView` ivar